log/notify.go

```diff
@@ -0,0 +1,56 @@
+package log
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"time"
+)
+
+type NotificationService struct {
```

Comment:
Can you make this an interface, and then have DiscordNotificationService implement it.

Reply:
It's been implemented but larger than expected https://github.com/Fesaa/iCalMerger/pull/4/commits/240e72d02fdb6287c73b663656f597b64218f062